app/views/layouts/application.html.erb

```diff
@@ -26,9 +26,9 @@
     <%= yield %>
     <!-- Add same styling as header //font & background colour. Container bottom of window with no padding -->
     <footer>
-      <div>
+      <div class='footer'>
         <!-- Add styling to center -->
-        <p>Made with love by the #April2017 <a href="http://makersacademy.com">Makers Academy</a> cohort</p>
+        <p>Made with love by the #April2017 <a class="footer_link" href="http://makersacademy.com">Makers Academy</a> cohort</p>
```

Comment:
Classes should use `-` as delimiters — IDs use `_`. Should have mentioned this, sorry!